### ISSUE TITLE
Incorrect resolution when directory and file have the same name

### ISSUE THREAD

**Issue by ForestBeaver** [benyap/resolve-tspaths#135]
Running into an issue when resolving a file that has a sibling directory of the same name.

Example repo: https://github.com/ForestBeaver/replace-ts-paths-same-name-example
```
npm install
npm run build

cat dist/controller/a.js

npm run replace

cat dist/controller/a.js

node dist/index.js
```

**Comment by benyap** [comment]
Hey @ForestBeaver, thanks for the bug report!

This issue should be resolved in the [0.7.2](https://github.com/benyap/resolve-tspaths/releases/tag/v0.7.2) release.

**Comment by ForestBeaver** [comment]
Thanks for the update @benyap!

I'm now getting this when trying to run with 0.7.2

```
Error: Cannot find module '~/version.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/beaverf/WebstormProjects/joy/product_service/node_modules/resolve-tspaths/dist/steps/createProgram.js:5:24)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
error Command failed with exit code 1.
```

**Comment by ForestBeaver** [comment]
It looks like `const version_json_1 = require("~/version.json");` isn't getting replaced in `dist/createProgram.js`

**Comment by benyap** [comment]
Yep that's my mistake haha - should be fixed in [0.7.3](https://github.com/benyap/resolve-tspaths/releases/tag/v0.7.3) :)
